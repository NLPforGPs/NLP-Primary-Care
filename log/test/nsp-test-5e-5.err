cpu-bind=MASK - bp1-gpu020, task 4294967295 4294967295 [0]: mask 0x1010 set

CondaValueError: prefix already exists: /user/home/zu20361/.conda/envs/NLP_GP

[nltk_data] Downloading package wordnet to
[nltk_data]     /user/home/zu20361/nltk_data...
[nltk_data]   Package wordnet is already up-to-date!
[nltk_data] Downloading package punkt to
[nltk_data]     /user/home/zu20361/nltk_data...
[nltk_data]   Package punkt is already up-to-date!
Some weights of the model checkpoint at microsoft/BiomedNLP-PubMedBERT-base-uncased-abstract were not used when initializing BertForNextSentencePrediction: ['cls.predictions.decoder.weight', 'cls.predictions.decoder.bias', 'cls.predictions.bias', 'cls.predictions.transform.dense.bias', 'cls.predictions.transform.LayerNorm.bias', 'cls.predictions.transform.dense.weight', 'cls.predictions.transform.LayerNorm.weight']
- This IS expected if you are initializing BertForNextSentencePrediction from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing BertForNextSentencePrediction from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
2021-12-08 13:45:27,811 - /user/home/zu20361/NLP-Primary-Care/./run_plms.py[line:150] - INFO: Predicting...
2021-12-08 13:45:28,163 - /user/home/zu20361/.conda/envs/NLP_GP/lib/python3.9/site-packages/datasets/builder.py[line:377] - WARNING: Using custom data configuration default-a2d0abb6534a5099
0 examples [00:00, ? examples/s]2021-12-08 13:45:28,379 - /user/home/zu20361/.cache/huggingface/modules/datasets_modules/datasets/transcript_evaldataset/4c048f6bc1501180d0c57dc4d2b8c0d9958946e6a0943b3c5b80a90f1d04b330/transcript_evaldataset.py[line:59] - INFO: Generating examples...
                                  0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [00:00<00:00, 295.96it/s]
2021-12-08 13:45:28,593 - /user/home/zu20361/NLP-Primary-Care/./run_plms.py[line:177] - INFO: NSP method
  0%|          | 0/1 [00:00<?, ?ba/s]100%|██████████| 1/1 [00:29<00:00, 29.67s/ba]100%|██████████| 1/1 [00:29<00:00, 29.68s/ba]

real	6m3.704s
user	3m52.334s
sys	1m31.848s
