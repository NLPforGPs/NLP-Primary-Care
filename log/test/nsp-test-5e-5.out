JOB ID: 243521
Working Directory: /user/home/zu20361/NLP-Primary-Care
Warning: you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies.  Conda may not use the correct pip to install your packages, and they may end up in the wrong place.  Please add an explicit pip dependency.  I'm adding one for you, but still nagging you.
Start Time: Wed Dec 8 13:44:37 GMT 2021
loading codes from PE...
Downloading and preparing dataset description_dataset/default (download: Unknown size, generated: 2.07 MiB, post-processed: Unknown size, total: 2.07 MiB) to /user/home/zu20361/.cache/huggingface/datasets/description_dataset/default-a2d0abb6534a5099/0.0.0/4c048f6bc1501180d0c57dc4d2b8c0d9958946e6a0943b3c5b80a90f1d04b330...
Dataset description_dataset downloaded and prepared to /user/home/zu20361/.cache/huggingface/datasets/description_dataset/default-a2d0abb6534a5099/0.0.0/4c048f6bc1501180d0c57dc4d2b8c0d9958946e6a0943b3c5b80a90f1d04b330. Subsequent calls will reuse this data.
classification_report:
                 precision    recall  f1-score   support

          blood       0.04      1.00      0.08         8
 cardiovascular       0.24      0.91      0.38        32
      digestive       0.45      0.82      0.58        44
            ear       0.61      1.00      0.76        11
      endocrine       0.16      0.92      0.27        24
            eye       0.18      0.80      0.30         5
         female       0.37      0.79      0.50        14
        general       0.06      1.00      0.11        14
           male       0.16      0.43      0.23         7
musculoskeletal       0.73      0.83      0.78        65
   neurological       0.13      0.80      0.23        20
      pregnancy       0.31      0.73      0.43        11
  psychological       0.36      0.90      0.51        50
    respiratory       0.46      0.95      0.62        37
           skin       0.52      0.72      0.61        32
     urological       0.20      0.94      0.34        18

      micro avg       0.24      0.86      0.37       392
      macro avg       0.31      0.85      0.42       392
   weighted avg       0.40      0.86      0.51       392
    samples avg       0.25      0.86      0.37       392

f1_score 0.4194085023649968
roc_auc 0.8728365690340871
End Time: Wed Dec 8 13:50:41 GMT 2021
